renv.lock

```diff
@@ -256,14 +256,14 @@
     },
     "ccao": {
       "Package": "ccao",
-      "Version": "1.2.2",
+      "Version": "1.3.0",
```

Comment:
I initially thought I had to update `ccao` due to an esoteric renv error that was complaining about it, but I think that was actually a red herring and the problem was that I wasn't restoring dependencies correctly. I left the update in as part of this PR, but I can downgrade it if you prefer.